flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamNode.java

```diff
@@ -113,6 +114,7 @@ public StreamNode(
             Class<? extends TaskInvokable> jobVertexClass) {
         this.id = id;
         this.operatorName = operatorName;
+        this.operatorDescription = operatorName;
```

Comment:
How about also providing `operatorDescription` as a constructor parameter? It doesn't look that different from `operatorName`. Then you can make the field `final` and get rid of the setter. While creating the streamNode, you would then simply either pass `transformation.getOperatorDescription()` or `operatorName` if the former is null.

Reply:
Using a separated setter can avoid touching all of the constructor of StreamNode, which could be quite a lot of code including helper method in StreamGraph and all of the TransformationTranslaters, and setting property of StreamNode in SimpleTransformationTranslator#configure is widely used. so I prefer init it by a setter in configure.
what do you think?

Reply:
Fair point. Let's go with that.